frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -214,7 +213,10 @@ export default {
           if (this.isAddingUser) {
```

Comment:
Minor point but on line 211 I would do an early return if not complete as it avoids having all your logic nested in the if block.

Reply:
Fixed